PixelHunter1995/Main.cs

```diff
@@ -6,6 +6,12 @@
 
 namespace PixelHunter1995
 {
+    public enum GameState
+    {
+        Menu,
+        Playing
+    }
```

Comment:
Use the state pattern: https://en.wikipedia.org/wiki/State_pattern

Reply:
Using state pattern now. Can probably be improved more by adding some kind of state handler.